clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -219,30 +219,24 @@ def drain_node(self, node_name: str, disable_eviction: bool) -> bool:
             return False
 
     def cordon_node(self, node_name: str) -> bool:
+        now = str(arrow.now().timestamp)
+        key = CLUSTERMAN_TERMINATION_TAINT_KEY
```

Comment:
nit, I think it's more readable if you reference the constant directly where it's used rather than renaming it locally.